src/test/java/core/basesyntax/UserServiceTest.java

```diff
@@ -36,7 +37,7 @@ public void getUserScore_exceptionExpected() {
     }
 
     @Test
-    public void getUserScore_validCase() {
+    public void getUserScore_validCase() throws UserNotFoundException {
```

Comment:
do we need to write `throws` for RuntimeExceptions?

Reply:
no